tests/integration/bond_test.py

```diff
@@ -501,3 +501,20 @@ def test_create_bond_with_mac(eth1_up, eth2_up):
         extra_iface_state={Interface.MAC: MAC0},
     ) as state:
         assertlib.assert_state_match(state)
+
+
+def test_bond_with_arp_ip_target(eth1_up, eth2_up):
+    with bond_interface(
+        name=BOND99,
+        slaves=[ETH1, ETH2],
+        extra_iface_state={
+            Bond.CONFIG_SUBTREE: {
+                Bond.MODE: BondMode.ACTIVE_BACKUP,
+                Bond.OPTIONS_SUBTREE: {
+                    "arp_interval": 1000,
+                    "arp_ip_target": "10.1.1.2,10.1.1.3",
```

Comment:
Actually, please do not use these IP addresses but RFC but addresses from https://www.kernel.org/doc/Documentation/networking/bonding.txt

Reply:
Done, thanks!